src/main/java/com/mewna/catnip/entity/message/MessageParse.java

```diff
@@ -0,0 +1,35 @@
+package com.mewna.catnip.entity.message;
+
+import lombok.Getter;
+
+import javax.annotation.CheckReturnValue;
+import javax.annotation.Nonnull;
+
+/**
+ * @author kjp12
+ * @see MessageOptions#parse()
+ * @since March 07, 2020
+ */
+public enum MessageParse {
```

Comment:
I think it would make more sense for this to be called `MentionParseFlag` or something similar?

Reply:
Changed as of 18731c0.